tests/python/README.md

```diff
@@ -20,14 +20,16 @@ the server calling REST API directly (as it done by users).
 ## How to run?
 **Initial steps**
 
-1. Follow [this guide](../../site/content/en/docs/api_sdk/sdk/developer-guide.md) to prepare
-   `cvat-sdk` and `cvat-cli` source code
-1. Install all necessary requirements before running REST API tests:
+1. On Debian/Ubuntu, make sure that your ```$USER``` is in ```docker``` group:
```

Comment:
Why do you use triple backticks here? Single backticks are enough for inline code.